src/overlays/actors/ovl_Bg_Jya_Ironobj/z_bg_jya_ironobj.c

```diff
@@ -89,15 +89,15 @@ static InitChainEntry sInitChain[] = {
 static CollisionHeader* sCollisionHeaders[] = { &gPillarCol, &gThroneCol };
 
 void BgJyaIronobj_InitCylinder(BgJyaIronobj* this, PlayState* play) {
-    ColliderCylinder* colCylinder = &this->colliderCylinder;
+    s32 pad;
 
-    Collider_InitCylinder(play, colCylinder);
-    Collider_SetCylinder(play, colCylinder, &this->dyna.actor, &sCylinderInit);
+    Collider_InitCylinder(play, &this->colliderCylinder);
```

Comment:
shouldn't it be `this->cylinderCollider`? the other renames in this PR seem to be changing `colliderXxxx` into `xxxxxCollider`.

Reply:
No this is consistent, "cylinder" is a type of collider so it goes suffix like the type `struct ColliderCylinder`
`xxxCollider` is when colliders are named something, like `bodyCollider` for example